public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -130,19 +142,45 @@ export const DataConnection = (props: { dataSource: string }) => {
     redirectToExplorerS3(dataSource);
   };
 
+  const dataSourceCardsText = isS3ConnectionWithLakeFormation
```

Comment:
it seems like this is also a conditional rendering for lakeformation only. So just wanna double confirm that no changes for s3 datasource connection.